src/NumberInput/NumberInput.svelte

```diff
@@ -137,16 +141,67 @@
     $$props["aria-label"] ||
     "Numeric input field with increment and decrement buttons";
 
+  //$: value && !!changeCaretPosition;
```

Comment:
If this is not needed, it should be removed.

Reply:
true. Removing